b2b_ecommerce/urls.py

```diff
@@ -1,17 +1,32 @@
 """URLs for business to business ecommerce"""
-from django.conf.urls import url
+from django.urls import path
 
-from b2b_ecommerce.views import B2BCheckoutView, B2BOrderFulfillmentView
+from b2b_ecommerce.views import (
+    B2BCheckoutView,
+    B2BOrderFulfillmentView,
+    B2BEnrollmentCodesView,
+    B2BOrderStatusView,
+)
 from mitxpro.views import index
 
 
 urlpatterns = [
-    url("^api/b2b/checkout/$", B2BCheckoutView.as_view(), name="b2b-checkout"),
-    url(
-        "^api/b2b/order_fulfillment/$",
+    path("api/b2b/checkout/", B2BCheckoutView.as_view(), name="b2b-checkout"),
+    path(
+        "api/b2b/order_fulfillment/",
         B2BOrderFulfillmentView.as_view(),
         name="b2b-order-fulfillment",
     ),
-    url("^ecommerce/bulk/$", index, name="bulk-enrollment-code"),
-    url("^ecommerce/bulk/receipt/$", index, name="bulk-enrollment-code-receipt"),
+    path(
+        "api/b2b/codes/<uuid:hash>/",
+        B2BEnrollmentCodesView.as_view(),
+        name="b2b-enrollment-codes",
+    ),
+    path(
+        "api/b2b/status/<uuid:hash>/",
```

Comment:
Nit: I just realized that the views for this route and the "codes" route above are both based on a specific `B2BOrder`. If we're being REST-y, these routes might make more sense to define as `"api/b2b/orders/<uuid:hash>/codes/"` and `"api/b2b/orders/<uuid:hash>/status/" respectively. The views could implement `get_object` to fetch the `B2BOrder`

Reply:
Just made the change, pretty straightforward